docs/memo.md

```diff
@@ -0,0 +1,189 @@
+# Memo
+
+## Dependency
+It is available in general package `"ru.tinkoff" %% "tofu" % Versions.tofu` and as standalone dependency `"ru.tinkoff" %% "tofu-memo" % Versions.tofu`
+
+## Functionality
+* A cache of a single value on access.
+* A cache of a values mapping on access.
+* TTL. Expired values are discarded on access.
+* Forced invalidation based on time.
+
+Missed:
```

Comment:
I guess something like `Not implemented` would sound a bit better than `Missed`

Reply:
Sorry but dont see a point.

Reply:
I dont like `not implemented` at all. It is like a promiss. Changed to neutral.